core/utils/poly-look/src/css/themes.css

```diff
@@ -0,0 +1,21 @@
+:root,
+.poly-theme {
+  --theme-font-base: var(--jost);
+  --theme-font-medium: var(--jost-medium);
+  color: var(--theme-secondary-color);
+  font-family: var(--jost);
+  font-weight: 400;
+  line-height: 120%;
+  letter-spacing: -0.01em;
+}
+
+:root,
+.poly-theme-light {
+  --theme-primary-color: var(--color-gray-50);
+  --theme-secondary-color: var(--color-indigo-950);
```

Comment:
OK, let me see if I got this straight. The color for the default theme is established by this theme? Shouldn't you maybe define that color independently of both and reuse it here? (Also, again "fallback" would be an option)